lib/mix/phoenix.ex

```diff
@@ -109,18 +110,20 @@ defmodule Mix.Phoenix do
         {_, _} -> false
        end)
     |> Enum.into(%{}, fn
-        {k, {:array, _}}      -> {k, []}
-        {k, :integer}         -> {k, 42}
-        {k, :float}           -> {k, "120.5"}
-        {k, :decimal}         -> {k, "120.5"}
-        {k, :boolean}         -> {k, true}
-        {k, :map}             -> {k, %{}}
-        {k, :text}            -> {k, "some content"}
-        {k, :date}            -> {k, "2010-04-17"}
-        {k, :time}            -> {k, "14:00:00"}
-        {k, :datetime}        -> {k, "2010-04-17 14:00:00"}
-        {k, :uuid}            -> {k, "7488a646-e31f-11e4-aace-600308960662"}
-        {k, _}                -> {k, "some content"}
+        {k, {:array, _}}         -> {k, []}
+        {k, :integer}            -> {k, 42}
+        {k, :float}              -> {k, "120.5"}
+        {k, :decimal}            -> {k, "120.5"}
+        {k, :boolean}            -> {k, true}
+        {k, :map}                -> {k, %{}}
+        {k, :text}               -> {k, "some content"}
+        {k, :date}               -> {k, "2010-04-17"}
+        {k, :time}               -> {k, "14:00:00"}
+        {k, :datetime}           -> {k, "2010-04-17 14:00:00"}
+        {k, :uuid}               -> {k, "7488a646-e31f-11e4-aace-600308960662"}
+        {k, {:unique, :integer}} -> {k, 42}
```

Comment:
What if someone uses unique:float or unique:uuid? I am wondering if we should pre-process the unique out before reaching this function, similar to how we do a couple lines above with references. :)